corehq/apps/locations/fixtures.py

```diff
@@ -79,6 +84,17 @@ def __call__(self, restore_user, version, last_sync=None, app=None):
         if not should_sync_locations(last_sync, all_locations, restore_user):
             return []
 
+        xml_nodes = []
+        for serializer in self.serializers:
+            xml_nodes.extend(serializer.get_xml_nodes(restore_user, all_locations))
```

Comment:
Is `xml_nodes` going to be a list of two fixture nodes?  What does it look like with both fixtures there?  Is there any danger of queries accessing both fixtures?